net-libs/udns/udns-0.6.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit edo toolchain-funcs
+
+DESCRIPTION="Async-capable DNS stub resolver library"
+HOMEPAGE="https://www.corpit.ru/mjt/udns.html"
+SRC_URI="https://www.corpit.ru/mjt/udns/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="ipv6 +tools"
+
+src_configure() {
+	# Uses non-standard configure script, econf doesn't work
+	CC="$(tc-getCC)" RANLIB="$(tc-getRANLIB)" edo ./configure $(use_enable ipv6)
+}
+
+src_compile() {
+	emake $(usex tools shared sharedlib)
+}
+
+src_install() {
+	dolib.so libudns.so.0
+	dosym libudns.so.0 /usr/$(get_libdir)/libudns.so
```

Comment:
Note that this pattern is pretty dangerous (see https://bugs.gentoo.org/862642). It's so much better if we can have a proper build system do it. I also don't generally love manual `src_install` as it so easily becomes out of date.

Reply:
I agree. This version has only a few fixes and does not change the existing situation on this point.

> I also don't generally love manual src_install as it so easily becomes out of date.

I won’t be too worried about this package ...